va/validation-authority.go

```diff
@@ -522,7 +530,7 @@ func (va *ValidationAuthorityImpl) checkCAAService(ctx context.Context, ident co
 		if prob.Type == probs.ServerInternalProblem {
 			prob.Detail = "Internal communication failure"
 		} else {
-			prob.Detail = err.Error()
+			prob.Detail = grpc.ErrorDesc(err)
```

Comment:
Can you add a comment explaining why this is here? It seems like any error that originates within the grpc system would naturally be a ServerInternalProblem, and should be turned into one by `bgrpc.CodeToProblem`.